content/articles/statistics/model-selection.md

```diff
@@ -24,60 +24,60 @@ We now walk through our two examples.
 
 ### $L_1$ optimization
 
-The analysis for $L_1$ optimization is similar to the above, but slightly more involved. We go through it quickly. The posterior with an $L_1$ prior is given by  
-$$ \tag{6}  
-P\left(\theta \vert \theta_0(\textbf{x})\right) \propto \exp\left( - \frac{(\theta - \theta_0)^2}{2 \sigma^2} - \Lambda \vert \theta \vert \right).  
-$$  
-Assuming for simplicity that $\hat{\theta} > 0$, the MAP value is now  
-$$ \tag{7}  
-\hat{\theta} = \begin{cases}  
-\theta_0 - \Lambda \sigma^2 & \text{if } \theta_0 - \Lambda \sigma^2 > 0 \\  
-0 & \text{else}.  
-\end{cases}  
-$$  
-The mean squared error of the estimator is  
-$$ \tag{8}  
-\langle (\hat{\theta} - \theta_*)^2 \rangle \equiv \int \frac{1}{\sqrt{2 \pi \sigma^2}}  
-\exp\left( - \frac{(\theta_0 - \theta_*)^2}{2 \sigma^2}\right) \left ( \hat{\theta} - \theta_* \right)^2 d \theta_0.  
-$$  
-This can be evaluated in terms of error functions. The optimal value of $\Lambda$ is obtained by differentiating the above. Doing this, one finds that it satisfies the equation  
-$$ \tag{9}  
-\exp\left( - \frac{(\tilde{\Lambda}- \tilde{\theta_*})^2}{2} \right ) + \sqrt{\frac{\pi}{2}} \tilde{\Lambda} \ \text{erfc}\left( \frac{\tilde{\Lambda} - \tilde{\theta_*}}{\sqrt{2}} \right ) = 0,  
-$$  
-where $\tilde{\Lambda} \equiv \sigma \Lambda$ and $\tilde{\theta_*} \equiv \theta_* / \sigma$. In general, the equation above must be solved numerically. However, in the case where $\theta_* \gg \sigma$ -- relevant when $N$ is large -- we can obtain a clean asymptotic solution. In this case, we have $\tilde{\theta_*} \gg 1$ and we expect $\Lambda$ small. This implies that the above equation can be approximated as  
-$$ \tag{10}  
-\exp\left( - \frac{\tilde{\theta_*}^2}{2} \right ) - \sqrt{2 \pi} \tilde{\Lambda} \sim 0.  
-$$  
-Solving gives  
-\begin{eqnarray} \tag{11}  
-\Lambda \sim \frac{1}{\sqrt{2 \pi \sigma^2}} \exp\left( - \frac{\theta_*^2}{2 \sigma^2} \right ) \sim \frac{\sqrt{N}}{\sqrt{2 \pi \sigma_1^2}} \exp\left( - \frac{N \theta_*^2}{2 \sigma_1^2} \right ).  
-\end{eqnarray}  
+The analysis for $L_1$ optimization is similar to the above, but slightly more involved. We go through it quickly. The posterior with an $L_1$ prior is given by
+$$ \tag{6}
+P\left(\theta \vert \theta_0(\textbf{x})\right) \propto \exp\left( - \frac{(\theta - \theta_0)^2}{2 \sigma^2} - \Lambda \vert \theta \vert \right).
+$$
+Assuming for simplicity that $\hat{\theta} > 0$, the MAP value is now
+$$ \tag{7}
+\hat{\theta} = \begin{cases}
+\theta_0 - \Lambda \sigma^2 & \text{if } \theta_0 - \Lambda \sigma^2 > 0 \\
+0 & \text{else}.
+\end{cases}
+$$
+The mean squared error of the estimator is
+$$ \tag{8}
+\langle (\hat{\theta} - \theta_*)^2 \rangle \equiv \int \frac{1}{\sqrt{2 \pi \sigma^2}}
+e^{ - \frac{(\theta_0 - \theta_*)^2}{2 \sigma^2}} \left ( \hat{\theta} - \theta_* \right)^2 d \theta_0.
```

Comment:
seems better to keep as `exp` given how complicated the argument is, i.e. a fraction.  just my opinion tho

Reply:
absoultely not